project.clj

```diff
@@ -44,7 +44,7 @@
                   :exclusions [prismatic/plumbing
                                potemkin
                                com.andrewmcveigh/cljs-time]]
-                 [threatgrid/ctim "1.0.11"
+                 [threatgrid/ctim "1.0.12-SNAPSHOT"
```

Comment:
This version of CTIM is not deployed on clojars with `lein deploy clojars`

Reply:
Done.